Files/MultilingualResources/Files.es-ES.xlf

```diff
@@ -2441,127 +2440,131 @@
         </trans-unit>
         <trans-unit id="NavToolbarColumnView.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>Column View</source>
-          <target state="new">Column View</target>
+          <target state="translated">Columnas</target>
         </trans-unit>
         <trans-unit id="NavToolbarColumnView.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>Column View</source>
-          <target state="new">Column View</target>
+          <target state="translated">Columnas</target>
         </trans-unit>
         <trans-unit id="BaseLayoutContextFlyoutColumn.Text" translate="yes" xml:space="preserve">
           <source>Column View</source>
-          <target state="new">Column View</target>
+          <target state="translated">Columnas</target>
         </trans-unit>
         <trans-unit id="PropertiesFilesFoldersAndLocationsCountString" translate="yes" xml:space="preserve">
           <source>{0:#,##0} files, {1:#,##0} folders from {2:#,##0} locations</source>
-          <target state="new">{0:#,##0} files, {1:#,##0} folders from {2:#,##0} locations</target>
+          <target state="translated">{0:#,##0} archivos, {1:#,##0} carpetas de {2:#,##0} ubicaciones</target>
         </trans-unit>
         <trans-unit id="ItemTypeLibrary" translate="yes" xml:space="preserve">
           <source>Library</source>
-          <target state="new">Library</target>
+          <target state="translated">Bibliotecas</target>
         </trans-unit>
         <trans-unit id="LibraryItemAutomation" translate="yes" xml:space="preserve">
           <source>Library</source>
-          <target state="new">Library</target>
+          <target state="translated">Bibliotecas</target>
         </trans-unit>
         <trans-unit id="PropertiesDialogTabLibrary.Content" translate="yes" xml:space="preserve">
           <source>Library</source>
-          <target state="new">Library</target>
+          <target state="translated">Bibliotecas</target>
         </trans-unit>
         <trans-unit id="LibraryLocations.Text" translate="yes" xml:space="preserve">
           <source>Locations:</source>
-          <target state="new">Locations:</target>
+          <target state="translated">Ubicaciones:</target>
         </trans-unit>
         <trans-unit id="LibraryLocationsButtonAdd.Label" translate="yes" xml:space="preserve">
```

Comment:
```suggestion
        <trans-unit id="LibraryLocationsButtonAdd.Label" translate="yes" xml:space="preserve">
```

I know you weren't the one who added these strings, but shouldn't this be `.Content`?

Reply:
surely yes, I can't test to see if that string is translated because the app crashes when trying to see the properties of the libraries

Reply:
It might be crashing because of that.

Reply:
![Captura de pantalla 2021-04-05 233249](https://user-images.githubusercontent.com/61259627/113655429-c7093780-9667-11eb-9fce-07025668a69f.png)

I changed them to "Content" in the xaml and the string resources, it still crashes

Reply:
`2021-04-05 23:38:54.2780|ERROR|Files.App|La aplicación llamó a una interfaz que se aplanó para un diferente subproceso. (Exception from HRESULT: 0x8001010E (RPC_E_WRONG_THREAD))|System.Exception: La aplicación llamó a una interfaz que se aplanó para un diferente subproceso. (Exception from HRESULT: 0x8001010E (RPC_E_WRONG_THREAD))
   at Windows.UI.Xaml.Controls.Image.put_Source(ImageSource value)
   at Files.UserControls.FileIcon.XamlBindingSetters.Set_Windows_UI_Xaml_Controls_Image_Source(Image obj, ImageSource value, String targetNullValue)
   at Files.UserControls.FileIcon.FileIcon_obj1_Bindings.Update_ViewModel_CustomIcon(SvgImageSource obj, Int32 phase)
   at Files.UserControls.FileIcon.FileIcon_obj1_Bindings.Update_ViewModel(SelectedItemsPropertiesViewModel obj, Int32 phase)
   at Files.UserControls.FileIcon.FileIcon_obj1_Bindings.Update_(FileIcon obj, Int32 phase)
   at Files.UserControls.FileIcon.FileIcon_obj1_Bindings.Update()
   at Files.UserControls.FileIcon.FileIcon_obj1_Bindings.Initialize()
   at Files.UserControls.FileIcon.FileIcon_obj1_Bindings.Loading(FrameworkElement src, Object data)
`

Reply:
It turns out that `label` is correct in this case.